### ISSUE TITLE
Module depth should be overridable

### ISSUE THREAD

**Issue by madAndroid** [ITV/domed-city#22]
We're currently hard coding the `-module-depth` to `1` .. which means if you have a set of modules that are going to make changes, you have restricted output, and it's difficult to tell what's going on:
```
Path: plans/my-amazing-plan.tf

~ module.users.aws_iam_group_membership.admins
    users.#:          "2" => "1"
    users.1234567890: "bob@example.com" => ""
    users.1234567890: "alice@example.com" => "alice@example.com"

~ module.infra.private_subnet_nat
    2 resource(s)
~ module.sec_groups.sg_web
    1 resource(s)

```

**Comment by bsnape** [comment]
I think it'll probably be good to have it verbose by default (because why not?) and configurable via an environment variable.

As long as we explain how and why this variable is set in the initial output then we're all good!

**Comment by madAndroid** [comment]
Sounds good to me! :)

**Comment by madAndroid** [comment]
fixed by #27
